worlds/stardew_valley/logic.py

```diff
@@ -968,11 +968,13 @@ def can_mine_to_floor(self, floor: int) -> StardewRule:
                  self.can_progress_easily_in_the_mines_from_floor(previous_previous_elevator)))
 
     def can_progress_in_the_skull_cavern_from_floor(self, floor: int) -> StardewRule:
-        tier = int(floor / 50)
+        tier = floor // 50
         rules = []
         weapon_rule = self.has_great_weapon()
+        mining_rule = self.has_skill_level(Skill.mining, min(10, tier + 5))
```

Comment:
I think this is supposed to go in the "if skills are progressive" condition, same as the combat level.

Any specific reason why you didn't use the same tier formula as for combat?